.github/ISSUE_TEMPLATE/membership-request.yml

```diff
@@ -0,0 +1,35 @@
+name: Membership nomination
+description: Nominate new etcd members
+labels:
+- type/feature
+- area/community
+body:
+- type: textarea
+  id: feature
+  attributes:
+    label: Who would you like to nominate?
+  validations:
+    required: true
+
+- id: requirements
+  type: checkboxes
+  attributes:
+    label: Requirements
+    options:
+    - label: I have reviewed the [community membership guidelines](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/community-membership.md)
+      required: true
+    - label: The members I am nominating have [enabled 2FA on their GitHub account](https://github.com/settings/security)
```

Comment:
I'm only not sure whether we should require from the sponsor, confirming whether formal requirements are full-filed by the nominee.  I would expect this to be self-declaration of nominee that happens after decision but probably before adding to the membership group.

Reply:
Fair point, I've taken the 2fa and dev mailing list tickboxes off for now as expectation is on member to self manage these and not put burden on sponsors to verify.

We can revisit this down the track if we shift from reviewer/maintainer nomination to self nomination.